content/docs/release-notes.md

```diff
@@ -2,7 +2,61 @@
 title: Release Notes
 ---
 
-## 2022-07-11
+## 2022-07-19 Storage release
+
+### What's new
+
+* Add WAL offloading to s3 on safekeepers. Separate task is launched for each timeline and stopped when timeline doesn't
```

Comment:
idea: prefix each change with the names of the 'components' if applies to.
e.g.
**Safekeeper**: verb something
**Pageserver**: verb something
**Proxy**: verb something
**Postgres Compute**: verb something

WDYT, @kelvich @lubennikovaav ?

Reply:
+1
It will look better. Some changes affect many components, so I am not sure how to label them..

Reply:
@lubennikovaav , list the components this touches, comma-separated

Reply:
could you add this, @lubennikovaav ?